src/applications/vre/28-1900/orientation/OrientationApp.jsx

```diff
@@ -54,10 +64,15 @@ const OrientationApp = props => {
             type="button"
             className="usa-button-primary"
           >
-            {step === 0 ? 'Start VR&E orientation' : 'Continue »'}
+            {/* eslint-disable-next-line no-nested-ternary */}
+            {step === 0
+              ? 'Start VR&E orientation slideshow'
+              : step < orientationSteps.length - 1
+                ? 'Next slide'
+                : 'Finish VR&E Orientation'}
           </button>
         </div>
-      </div>
+      </article>
       {formStartControl && (
         <div className="vads-u-padding--3 vads-u-background-color--gray-lightest">
```

Comment:
I did look at the overall structure of the page, but since the above wrapper is now an `article`, would it make sense to also make this an `article`?